azure-pipelines/cd.yml

```diff
@@ -1,64 +1,179 @@
-trigger:
-  tags:
-    include:
-    - v*
-
+# Manual trigger only
+trigger: none
 pr: none
 
-strategy:
-  matrix:
-    linux:
-      imageName: 'ubuntu-16.04'
-    mac:
-      imageName: 'macos-10.14'
-    windows:
-      imageName: 'vs2017-win2016'
-
-pool:
-  vmImage: $(imageName)
+parameters:
+- name: variableGroup
+  displayName: 'Pipeline variable group'
+  type: string
+  default: 'releaseVariables'
```

Comment:
We could not have this variableGroup param at all and associate through ADO pipeline config.